spidermon/contrib/scrapy/extensions.py

```diff
@@ -157,8 +174,20 @@ def item_scraped(self, item, response, spider):
         skip_none_values = spider.crawler.settings.getbool(
             "SPIDERMON_FIELD_COVERAGE_SKIP_NONE", False
         )
+        skip_fields_without_rules = spider.crawler.settings.getbool(
+            "SPIDERMON_FIELD_COVERAGE_SKIP_WITHOUT_FIELD_COVERAGE_RULES", False
+        )
+        field_coverage_rules = spider.crawler.settings.get(
+            "SPIDERMON_FIELD_COVERAGE_RULES", {}
+        )
         self.crawler.stats.inc_value("spidermon_item_scraped_count")
-        self._count_item(item, skip_none_values)
+
+        if skip_fields_without_rules and not field_coverage_rules:
+            return
+
+        self._count_item(
+            item, skip_none_values, field_coverage_rules, skip_fields_without_rules
+        )
```

Comment:
We could make this cleaner by first defining `skip_fields` as `None` if `skip_fields_without_rules` is `None` or as the set of fields to ignore otherwise, hence only passing 1 variable forward.